ingestion_server/Pipfile

```diff
@@ -5,7 +5,7 @@ verify_ssl = true
 
 [dev-packages]
 ipython = "~=8.14"
-pycodestyle = "~=2.10"
+pycodestyle = "~=2.11"
```

Comment:
Ideally, this should not even be a package. It should be a pre-commit hook.

Reply:
More info: `flake8` already includes `pycodestyle`. So this is redundant and can safely be removed.